src/main.rs

```diff
@@ -19,12 +19,16 @@ Command-line interface for Proton
 Usage:
   ./proton init <folder> <root-public-key>
```

Comment:
Is there such a thing as argv[0] to get the exact program name instead of hardcoding one?